contracts/NonFungibleToken.cdc

```diff
@@ -194,7 +197,16 @@ access(all) contract interface NonFungibleToken: ViewResolver {
 
         /// Gets the amount of NFTs stored in the collection
         /// @return An integer indicating the size of the collection
-        access(all) view fun getLength(): Int
+        access(all) view fun getLength(): Int {
+            return self.ownedNFTs.length
+        }
+
+        /// Returns an iterator that allows callers to iterate
+        /// through the list of owned NFT IDs in a collection
+        /// without having to load the entire list first
+        access(all) fun forEachID(_ f: fun (UInt64): Bool): Void {
```

Comment:
```suggestion
        access(all) fun forEachID(_ f: fun (UInt64): Bool) {
```

Reply:
I'm going to keep the `Void` here because I think it makes it more clear